projects/wallet-template/src/containers/WalletPopup/pages/Accounts/AccountDetails.tsx

```diff
@@ -0,0 +1,60 @@
+import { ArrowLeft, ClipboardCheck, ClipboardCopyIcon } from "lucide-react"
+import * as clipboard from "@zag-js/clipboard"
+import { useMachine, normalizeProps } from "@zag-js/react"
+import React, { useId } from "react"
+import { useParams, useNavigate } from "react-router-dom"
+
+export const AccountDetails: React.FC = () => {
+  const navigate = useNavigate()
+  const { accountId } = useParams<{ accountId: string }>()
+  const [state, send] = useMachine(
+    clipboard.machine({
+      id: useId(),
+      value: accountId,
+    }),
+  )
+
+  const api = clipboard.connect(state, send, normalizeProps)
+
+  if (!accountId) {
+    return null
+  }
```

Comment:
this seems to be redundant because the react-router will not match a route with an empty `accountId`